source/administration/config-settings.rst

```diff
@@ -2083,6 +2083,31 @@ A list of all the machines in the cluster, separated by commas, for example, ``[
 | This feature’s ``config.json`` setting is ``"InterNodeUrls": []`` with string input.                                                                                        |
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+________________________________________________________________________________________________________________________________________________________________________
+
+Performance Monitoring (Beta)
+```````````````````````````
+*Available in Enterprise Edition E20*
+
+Enable Performance Monitoring
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**True**: When true, Mattermost will enable performance monitoring collection and profiling. Please see `documentation <https://docs.mattermost.com/deployment/metrics.html>`_ to learn more about configuring performance monitoring for Mattermost.
```

Comment:
For consistency we should remove the "When true," as we don't do that on most other settings